adminapi/cmduser.py

```diff
@@ -1,7 +1,26 @@
 import os
+from os.path import isfile
 import pwd
 
 
+def get_auth_token():
+    user = get_user()
+    config_file = os.path.join(user.pw_dir, '.adminapirc')
+    if isfile(config_file):
```

Comment:
In a next move, you should try to reduce indentions.

Reply:
We can.